.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
```suggestion
charset = utf-8
indent_style = space
indent_size = 2
insert_final_newline = true
trim_trailing_whitespace = true
```

Reply:
The diffs on GitHub were a bit confusing. I hope I got all of it right.